Sources/prefrence/ALApplozicSettings.m

```diff
@@ -1906,4 +1906,14 @@ + (NSString *)getSupportContactUserId {
     return [userDefaults valueForKey:AL_SUPPORT_CONTACT_USER_ID];
 }
 
++ (void) enableAgentApConfiguration:(BOOL)flag {
```

Comment:
Instead of `agentAppConfiguration` we can rename it to `enableOfflineStatusConfig` or something. It will be more specific.